modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -4,33 +4,39 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: schedules, cloudhub, polling
 
-The CloudHub console includes a *Schedules* tab in the left navigation menu where you can view and control any polling elements within the flows of your deployed applications. CloudHub automatically detects the polling elements from within your flows and lists them on the *Schedules* tab. On this tab, you can select the polling elements and enable or disable them, change the polling frequency, or run them immediately – all during runtime, without making any changes in the underlying application.
+The *Schedules* tab in the left navigation menu of the CloudHub console enables you to view and control any polling elements within the flows of your deployed applications. On this tab, you can manage polling elements, change the polling frequency, or run them immediately – all during runtime, without making any changes in the underlying application.
 
 == Use Cases
 
 The *Schedules* tab is useful in the following scenarios:
 
-* You have a job that runs nightly but you want to trigger it right now so you can debug it. You want to do this in runtime so you don’t need to update the schedule in your underlying application.
+* You have a job that runs nightly but you want to debug it now.
++
+You can trigger it while it's running without updating the schedule in your underlying application.
 * You want to change the schedules in your application at runtime without redeploying.
 * You want to disable a scheduled job while one of the applications you are connecting to is undergoing maintenance.
 * You want runtime visibility into the schedules in your application.
 
 == Considerations
 
+When using the *Schedules* tab, consider the following:
+
 * Schedulers are managed by CloudHub and are triggered in only one worker even if more are available.
-* Setting a polling Schedule for less than a 10-second frequency between calls is not recommended.
+* Setting a polling schedule for less than a 10-second frequency between calls is not recommended.
 * If a scheduled job is not triggered one or more times because the application is down, CloudHub triggers the job as soon as the application restarts.
-* CloudHub scheduler reads a schedule job configuration every time it runs. This means that if you update the schedule time, the change is not immediate. The job executes as it was initially configured before the scheduler picks up the next setting for the next time it runs. To apply your changes immediately, run your jobs after every schedule update.
+* CloudHub scheduler reads a schedule job configuration every time it runs.
```

Comment:
Is this wording correct, "a schedule job configuration"? Is it accurate to use only "a job configuration"?